railties/lib/rails/generators/rails/app/templates/config/boot.rb.tt

```diff
@@ -1,13 +1,13 @@
-<% unless options.skip_spring? -%>
-begin
+# frozen_string_literal: true
+
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
+<% if spring_install? -%>
+
+if !defined?(Spring) && [nil, "development", "test"].include?(ENV["RAILS_ENV"])
   load File.expand_path("../bin/spring", __dir__)
-rescue LoadError => e
-  raise unless e.path.end_with?("/bin/spring")
```

Comment:
Hey, @eugeneius  why this has been removed? By default rails do not include spring into test env, so on CI with rails env test is not available. And before these changes, CI skipped spring. Now we have an error. I need to understand the reason in order to create a hot fix for that change.

Reply:
@pftg Please see #40913.  Sorry for the trouble!